python_benchmarks/qr_codes/qr.py

```diff
@@ -162,11 +181,11 @@ def main():
                         default="./", action="store",
                         dest="model_path")
     parser.add_argument("-a", "--accuracy", help="input accuracy", default="20", action="store", dest="accuracy",
-                        type=int)
+                        type=float)
     parser.add_argument("-alg", "--algorithm", help="QR detect algorithm", default="opencv", action="store",
```

Comment:
It makes sense to extend QrCode class with read/write methods to dump settings as it's done for cv::Algorithm implementations and feed config with command line. It allows to select implementations from objdetect and try custom settings.

Reply:
I think it's better to add to the new PR.